starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express')
+
+const celebrityModel = require('../models/celebrity')
+
+const router = express.Router()
+
+
+router.get('/celebrities', (req,res, next)=>{
+    celebrityModel.find()
+    .then((celebrity) =>{
```

Comment:
this query will fetch you several elements so naming your parameter "celebrities" would be more accurate.